chapters/testing.Rmd

```diff
@@ -1165,6 +1166,69 @@ def total(values):
 Write unit tests to check that the `assert` statements in the previous question's `total` function
 do what they're supposed to.
 
+### Test error handling {#testing-ex-error-handling}
+
+In Chapter \@ref(errors), we added some functionality to `collate.py` in order to handle
```

Comment:
"we modified `collate.py` to handle" rather than "we added some functionality to `collate.py` in order to handle"